internal/eventlog/eventlog.go

```diff
@@ -0,0 +1,35 @@
+package eventlog
```

Comment:
We may want to rename this package to just `event`.

Reply:
I choose the name based on https://brandur.org/fragments/go-no-common-nouns.